Arrt/View/ModelEditor/Stats/StatsGraph.cpp

```diff
@@ -390,7 +408,7 @@ void StatsGraph::setHighlightX(std::optional<float> x)
                             m_highlightedPoints.clear();
                             m_highlightedX = chosen.x();
                         }
-                        m_highlightedPoints.push_back({m_infos[valIdx].m_color, QString("%1: %2%3").arg(m_infos[valIdx].m_name).arg(toString(chosen.y())).arg(m_infos[valIdx].m_units), chosen});
+                        m_highlightedPoints.push_back({m_infos[valIdx].m_color, QString("%1: %2%3").arg(m_infos[valIdx].m_name).arg(toHumanReadableString(chosen.y())).arg(m_infos[valIdx].m_units), chosen});
```

Comment:
I wouldn't simplify the highlighted values on the tooltip (so we don't have approximation). I would just simplify the Y labels

Reply:
Discussed offline, it looks a lot neater to have the simplified string in the tooltip too.